client/lib/analytics/README.md

```diff
@@ -122,3 +139,16 @@ If we had instead used `calypso_add_cart_product` and `calypso_remove_cart_produ
 Finally, for consistency, the verb at the end should be in the form `add`, `remove`, `view`, `click`, etc and _not_ `adds`, `added`, `adding`, etc.
 
 With the exception of separating tokens with underscores, these rules do not apply to property names. `coupon_code` is perfectly fine.
+
+## Page Views Paths Conventions
+
+There are currently no enforced rules for page views paths reporting but, where possible, they should be normalized and the variables values replaced by placeholders.
+
+E.g. the path `/comments/all/example.wordpress.com/1234` should be reported as `/comments/all/:site/:post_id`.
+
+As a rule of thumb:
+
+- The **site fragment** should be reported simply as `:site`.
+- **IDs** should be reported as `:element_id` (notice the snake case).
+- **Non-enumerable variables** (e.g. the domain name in `/domains/manage/:domain/edit/:site`) should be reported without unnecessary qualifiers.
```

Comment:
Maybe a different word for `enumerable`? I tend to associate this with what I can loop over in an object or array. Maybe infinite vs finite, or known set (views) vs unknown values (domains, ids, etc)

We may want to grab a few folks to see what reads better :)

Reply:
This one is tricky to phrase, since even we're not always getting it right :D Maybe not known in code?
See also https://github.com/Automattic/wp-calypso/pull/23904#issuecomment-378699561